### PR TITLE
Windows Services updates

### LINKED ISSUE #9226: New doc and sample: Convert a project into a Windows Service
From discussion on https://github.com/aspnet/Docs/pull/9169 ...

@blowdart ...

> Local System? Local System is high privilege. Shouldn't we be recommending creating a specific user here? ... local system? That can't be a real recommendation here.

@Tratcher ...

> What account would you recommend?

@blowdart ...

> A custom one created for the app. Not a built in account at all.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: b0bd8cb8-7e28-8c4d-a550-3aea8f3ab4dd
* Version Independent ID: 4b60d24e-ef05-15e5-10b4-483094081bf3
* Content: [Host ASP.NET Core in a Windows Service](https://docs.microsoft.com/en-us/aspnet/core/host-and-deploy/windows-service?view=aspnetcore-2.1)
* Content Source: [aspnetcore/host-and-deploy/windows-service.md](https://github.com/aspnet/Docs/blob/master/aspnetcore/host-and-deploy/windows-service.md)
* Product: **aspnet-core**
* GitHub Login: @guardrex
* Microsoft Alias: **tdykstra**

### PR DESCRIPTION
**Pull request by guardrex**
Fixes #9226 

[Internal Review Topic (Convert a project into a Windows Service section)](https://review.docs.microsoft.com/en-us/aspnet/core/host-and-deploy/windows-service?view=aspnetcore-2.1&branch=pr-en-us-9337#convert-a-project-into-a-windows-service)

* Drops 1.x and 2.0 coverage. Versions whole topic to 2.1 or later.
* Upgrades sample app to 2.2 preview3. Added to the 2.2 RTM update tracking issue: #8518
* Drops *wwwroot* assets. Uses new `<style>` CSS styles.
* 🎪 **User account**: Instructs the user on how to create a user account for the service, assign permissions, and set up the service to use the account. The only permissions granted to the user account are read, write, and execute on the service's deployment folder.

Note: The topic is currently still relying on SCD deployment. The updates for FDD/FDE are coming later on another issue.

### CODE REVIEW

**Comment by guardrex** [comment]
> One thing it might be worth calling out is where you say {domain} if you're not domain joined it should be the local machine?

Yes ... I **did NOT** do a good job on that. I'll get it on the next commit. Thanks.

**Comment by guardrex** [comment]
@scottaddie No rush on this.

File: aspnetcore/host-and-deploy/windows-service.md [hunk 1]
```diff
@@ -71,56 +46,90 @@ The following minimum changes are required to set up an existing ASP.NET Core pr
 
    * Call [UseContentRoot](xref:fundamentals/host/web-host#content-root) and use a path to the app's published location instead of `Directory.GetCurrentDirectory()`.
 
-     ::: moniker range=">= aspnetcore-2.0"
-
      [!code-csharp[](windows-service/samples/2.x/AspNetCoreService/Program.cs?name=ServiceOnly&highlight=8-9,16)]
 
-     ::: moniker-end
+1. Publish the app. Use [dotnet publish](/dotnet/articles/core/tools/dotnet-publish) or a [Visual Studio publish profile](xref:host-and-deploy/visual-studio-publish-profiles). When using a Visual Studio, select the **FolderProfile** and configure the **Target Location** before selecting the **Publish** button.
```
Comment (Rick-Anderson) [inline aspnetcore/host-and-deploy/windows-service.md]: Is there a reason you didn't write it as:

`Publish the app using [dotnet publish](/dotnet/articles/core/tools/dotnet-publish) or a [Visual Studio publish profile](xref:host-and-deploy/visual-studio-publish-profiles).`

File: aspnetcore/host-and-deploy/windows-service.md [hunk 2]
```diff
@@ -71,56 +46,90 @@ The following minimum changes are required to set up an existing ASP.NET Core pr
...
-     ::: moniker-end
+1. Create a user account for the service using the `net user` command:
 
-1. Publish the app. Use [dotnet publish](/dotnet/articles/core/tools/dotnet-publish) or a [Visual Studio publish profile](xref:host-and-deploy/visual-studio-publish-profiles). When using a Visual Studio, select the **FolderProfile**.
+   ```console
+   net user {USER ACCOUNT} {PASSWORD} /add
+   ```
 
-   To publish the sample app using command-line interface (CLI) tools, run the [dotnet publish](/dotnet/core/tools/dotnet-publish) command at a command prompt from the project folder. The RID must be specified in the `<RuntimeIdenfifier>` (or `<RuntimeIdentifiers>`) property of the project file. In the following example, the app is published in Release configuration for the `win7-x64` runtime:
+   For the sample app, create a user account with the name `ServiceUser` and a password (for example, `Passw0rd$`):
```
Comment (Rick-Anderson) [inline aspnetcore/host-and-deploy/windows-service.md]: @blowdart 	do we really want to suggest passwords customers will copy/paste?

Comment (guardrex) [reply aspnetcore/host-and-deploy/windows-service.md]: I removed it and tell the user to replace `{PASSWORD}` with the user account's password in these spots.

**Pull request merged**